### PR TITLE
Fix issue with proxied clientlib

### PR DESCRIPTION
**Pull request by pcastelog**

### CODE REVIEW

**Comment by kwin** [comment]
@pcastelog Thanks for the PR. Can you add a changelog entry and a test for this?

**Comment by henrykuijpers** [comment]
@kwin  I think it is very important to get this issue pushed in very quickly. The entire functionality is broken now (for us, mainly on author, since we have no /etc/map entries there).

**Comment by pcastelog** [comment]
I added the requested changes
- Test for the case of resolving a clientlib with resourceResolver
- Changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 ## Unreleased ([details][unreleased changes details])
 <!-- Keep this up to date! After a release, change the tag name to the latest release -->
 [unreleased changes details]: https://github.com/Adobe-Consulting-Services/acs-aem-commons/compare/acs-aem-commons-5.0.4...HEAD
+### Fixed
```
Comment (kwin) [inline CHANGELOG.md:11]: there should be only one section called "Fixed". Can you please just extend the existing section in line 12?

**Review by kwin — APPROVED** [review]

**Pull request merged**